src/coreclr/src/debug/daccess/request.cpp

```diff
@@ -4343,13 +4343,6 @@ HRESULT ClrDataAccess::IsRCWDCOMProxy(CLRDATA_ADDRESS rcwAddr, BOOL* isDCOMProxy
     *isDCOMProxy = FALSE;
 
 #ifdef FEATURE_COMINTEROP
```

Comment:
Would it make more sense to remove the #ifdef?

Reply:
@davmason I can do that. I didn't change it simple to ensure the behavior of the API is the same when COM isn't supported. I assume SOS would indicate "command failed" now and if I change to `S_OK` then would say "Not a COM proxy", but COM isn't support anyways. Really is up to you, I can remove it and always return `E_NOTIMPL` or `S_OK`.

Reply:
For some reason I was thinking that FEATURE_COMINTEROP was always defined, but of course it isn't on linux/mac. So I think the way you have it makes sense.